include/player.h

```diff
@@ -1,9 +1,35 @@
 #ifndef PLAYER_H
 #define PLAYER_H
+#include <SFML/Graphics.hpp>
+#include <vector>
 #include "shooter.h"
+#include "platform.h"
 
 class Player : public Shooter {
-
+    private:
+        sf::Texture texture;
+        float velocityX;
+        float velocityY;
+        float xpos;
+        float ypos;
+        float maxJumpHeight;
```

Comment:
Will this be a constant value? If it is, then we should probably define it as a const